test/parallel/test-http-outgoing-internal-headernames.js

```diff
@@ -0,0 +1,22 @@
+// Flags: --expose-internals
+'use strict';
+const common = require('../common');
+
+const { OutgoingMessage } = require('http');
+
+const warn = 'OutgoingMessage.prototype._headerNames is deprecated';
+common.expectWarning('DeprecationWarning', warn, 'DEP0066');
```

Comment:
We have enabled this warning in both getter and setter. But, the warning will be emitted only once. We may have to write different tests to actually test emission of this warning for both the cases. Am I missing something?

Reply:
Yes, that's right. This will need either two test files, one for the setter and one for the getter, or else the test file will have to test by spawning its own Node.js instances, one instance for the setter and one for the getter.

Reply:
OK, done. Will re-run CI...